### PR TITLE
Release candidate v0.1.0-beta.8

### PR DESCRIPTION
**Pull request by AndyTitu**

### CODE REVIEW

File: src/onepassword/secrets.py [hunk 1]
```diff
@@ -3,13 +3,10 @@
 
 
 class Secrets:
-    """Contains all operations the SDK client can perform on 1Password secrets."""
-
```
Comment (hculea) [inline src/onepassword/secrets.py:7]: Let's add all the deleted comments back @AndyTitu.

Comment (hculea) [reply src/onepassword/secrets.py:7]: Actually, we can simply revert all the changes outside lib and setup.py, right?

Comment (AndyTitu) [reply src/onepassword/secrets.py]: Yes, we can refert `items.py` and `secrets.py`. I reverted them

**Comment by hculea** [comment]
Closing in favour of https://github.com/1Password/onepassword-sdk-python/pull/44

**Pull request closed without merging**